### PR TITLE
Change generated bg for #6a737d to #444

### LINKED ISSUE #858: Default issue/pr status label is too dark
* **Browser**: chrome
* **Operating System**: win10
* **Link to page with the issue**: https://github.com/StylishThemes/GitHub-Dark/pull/857
* **Screenshot**:
image image

I tried adding `.State` to the `background: #444`  rule, and adding a new one that used `background-color`, but because the color variants are being auto generated they show up earlier in the style and get overwritten by the new default color rule.

### PR DESCRIPTION
**Pull request by xt0rted**
This fixes #858 as well as another issue where the `required` status check label wasn't visible.

image

image

I'm not sure where the other selectors for this rule are used so I can't verify this didn't break anything. Does anyone know where these may be used?

https://github.com/StylishThemes/GitHub-Dark/blob/52bf73ee256c84f1d1d22249b99238e75375b461/github-dark.css#L162-L171

### CODE REVIEW

**Comment by the-j0k3r** [comment]
Couldnt we go with a shade darker, say `#444`? Otherwise LGTM

**Comment by silverwind** [comment]
Yes, a bit darker would be nice.

**Comment by silverwind** [comment]
> Does anyone know where these may be used?

From that list, it appears mostly icons inside discussions. Maybe you can also check for potential duplicate selectors in the style from that list, and potentially remove them.

**Comment by xt0rted** [comment]
I went with `#555` because it's already being used for the branch status and merge status icons.

image

A couple of the selectors in the autogenerated rule were already being styled as `#555` in another rule so they were removed, you can see that in the diff. I'll take a look at the rest to see what I can find.

**Comment by xt0rted** [comment]
Here's one of the mystery spots (`.discussion-item-head_ref_deleted .discussion-item-icon`).

image image

**Comment by silverwind** [comment]
Make all of them #444, ideally through generator rules only.

**Comment by xt0rted** [comment]
Will do.

Found a few more spots that are part of this.

image

Both of those icons use the same markup and are styled by `.discussion-item-icon` which has a background of `#333` right now.

```html
<span class="discussion-item-icon">
  <svg class="octicon octicon-repo-force-push" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M10 9H8v7H6V9H4l2.25-3H4l3-4 3 4H7.75L10 9zm1-9H1C.45 0 0 .45 0 1v12c0 .55.45 1 1 1h4v-1H1v-2h4v-1H2V1h9v9H9v1h2v2H9v1h2c.55 0 1-.45 1-1V1c0-.55-.45-1-1-1z"></path></svg>
</span>
```

The required label also shows up under the protected branches rules which is against a `#181818` background.

image

```html
<div class="listgroup listgroup-sm automated-check-options">
  <div class="listgroup-item listgroup-header">
    Status checks found in the last week for this repository
  </div>
  <div class="listgroup-overflow">
      <div class="listgroup-item js-protected-branches-item">
        <label>
          <input type="checkbox" class="js-required-status-checkbox" name="contexts[]" value="Block Fixup Commits">
          Block Fixup Commits
          <span class="label Label--gray-darker float-right js-required-status-badge  d-none">Required</span>
        </label>
      </div>
      <div class="listgroup-item js-protected-branches-item">
        <label>
          <input type="checkbox" class="js-required-status-checkbox" name="contexts[]" value="default">
          default
          <span class="label Label--gray-darker float-right js-required-status-badge  d-none">Required</span>
        </label>
      </div>
      <div class="listgroup-item js-protected-branches-item">
        <label>
          <input type="checkbox" class="js-required-status-checkbox" name="contexts[]" value="fixup-action" checked="">
          fixup-action
          <span class="label Label--gray-darker float-right js-required-status-badge ">Required</span>
        </label>
      </div>
  </div>
</div>
```

**Comment by the-j0k3r** [comment]
Ide probably rebase this on master just to get rid of unrelated generated stuff which https://github.com/StylishThemes/GitHub-Dark/commit/bd70b73c9bbd38f5c1290e2765b213c712ea7d33 took care of.

Once this is in we could do another rel too ;)

**Comment by xt0rted** [comment]
This is what the PR timeline & related icons look like with `#444` as the background color.

image image

image

I'm still testing to see if I can find any other spots that are effected but so far it seems good.

File: github-dark.css [hunk 1]
```diff
@@ -3010,17 +3010,15 @@
...
-  .manage-repo-access-wrapper:before, .counter, .Counter, .block-diff-neutral {
+  .manage-repo-access-wrapper:before, .counter, .Counter, .block-diff-neutral,
+  .discussion-item-icon {
     background: #444 !important;
   }
   .switcher > span:after, .list-module a:hover, .section-nav .active a,
   #dashboard .filter-bar li a.filter_selected:after, .state-widget .behind .bar,
   .settings-email .label:not(.default), .key, .date:after,
   .branch-a-b-count .bar, .branch-a-b-count .meter,
-  .branch-action-state-dirty .branch-action-icon,
```
Comment (xt0rted) [inline github-dark.css]: These are part of the generated rule for this color.

**Comment by xt0rted** [comment]
Most of the selectors in this rule are for the action & status icons in the timeline (what's in my screenshots above). The other two selectors (`.issues-reset-query-icon, .projects-reset-query-icon`) are the `X` icon to remove the search filter:
- https://github.com/StylishThemes/GitHub-Dark/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+test
- https://github.com/StylishThemes/GitHub-Dark/projects?utf8=%E2%9C%93&query=is%3Aopen+test

The icon for project search is broke, incoming fix once this is merged.

**Pull request merged**